ts/util/text-input.ts

```diff
@@ -58,9 +82,30 @@ export class TextInput {
     }
   }
 
-  public getTag() {
+  private async loadAutocomplete(queryUrl: string) {
+    const val = this.textField.value;
+    this.datalist.replaceChildren(); // empties autocomplete
+    if (val.length < 2) {
+      // results are only displayed for input > **2** characters
```

Comment:
Can change to work after 1 character. It's really fast, so not a problem. We can adjust in the future if necessary